src/tests/dotnet-trace/CollectCommandFunctionalTests.cs

```diff
@@ -130,71 +130,71 @@ public static IEnumerable<object[]> BasicCases()
                 ExpectProvidersWithMessages(
                     new[]
                     {
-                        "No profile or providers specified, defaulting to trace profile 'cpu-sampling'"
+                        "No profile or providers specified, defaulting to trace profiles 'dotnet-common' + 'dotnet-sampled-thread-time'."
                     },
-                    FormatProvider("Microsoft-DotNETCore-SampleProfiler", "0000F00000000000", "Informational", 4, "--profile"),
-                    FormatProvider("Microsoft-Windows-DotNETRuntime", "00000014C14FCCBD", "Informational", 4, "--profile"))
+                    FormatProvider("Microsoft-Windows-DotNETRuntime", "000000100003801D", "Informational", 4, "--profile"),
+                    FormatProvider("Microsoft-DotNETCore-SampleProfiler", "0000F00000000000", "Informational", 4, "--profile"))
             };
 
             yield return new object[]
             {
-                new CollectArgs(providers: "Foo:0x1:4"),
+                new CollectArgs(providers: new[] { "Foo:0x1:4" }),
                 ExpectProviders(
                     FormatProvider("Foo", "0000000000000001", "Informational", 4, "--providers"))
             };
 
             yield return new object[]
             {
-                new CollectArgs(providers: "Foo:0x1:4,Bar:0x2:4"),
+                new CollectArgs(providers: new[] { "Foo:0x1:4", "Bar:0x2:4" }),
                 ExpectProviders(
                     FormatProvider("Foo", "0000000000000001", "Informational", 4, "--providers"),
                     FormatProvider("Bar", "0000000000000002", "Informational", 4, "--providers"))
             };
 
             yield return new object[]
             {
-                new CollectArgs(profile: "cpu-sampling"),
```

Comment:
This test case would be interesting to keep I think although the expected behavior is now different. There is a decent chance users will try to use this profile because it used to be valid and now hopefully they will get a helpful error message.